dev/tasks/tasks.yml

```diff
@@ -1542,8 +1542,8 @@ tasks:
       image: conda-python-hdfs
 {% endfor %}
 
-{% for python_version, spark_version, test_pyarrow_only, numpy_version, jdk_version in [("3.8", "v3.5.0", "false", "latest", "8"),
-                                                                                        ("3.10", "v3.5.0", "false", "1.23", "8"),
+{% for python_version, spark_version, test_pyarrow_only, numpy_version, jdk_version in [("3.8", "v3.5.0", "false", "latest", "11"),
+                                                                                        ("3.10", "v3.5.0", "false", "1.23", "11"),
```

Comment:
You can delete these. We can't test against Spark 3.X because its a Java 8 project. Spark 4.X supports Java 17+.

Reply:
Does it mean that master is the only spark version we will test again for a while?

Reply:
Yep, Spark 4.0 has not yet officially released. We'll add it here once its available.